src/components/BorrowerProfile/BorrowerBusinessDetails.vue

```diff
@@ -36,6 +39,7 @@
 					tw-inline-flex tw-items-center tw-justify-center
 					tw-mt-2 tw-mr-2
 					tw-h-6 tw-w-6"
+			:data-testid="`bp-direct-business-social-links-${index}`"
```

Comment:
If you wanted some additional specificity you could use `link.type` in place of `index` here to show "facebook" or "twitter" etc...

Reply:
I did like the idea of using link.type, but after careful consideration I think using index is best.  It is just easier to iterate through a loop to see which, if any, social link exist.  We can either get the href attribute for the element or alt attribute for the child svg to determine which site it is.